job_creator.php

```diff
@@ -71,8 +76,21 @@ public function createJob(int $phpIndex, array $opts): array
         ];
         return array_merge($default, $opts);
     }
+
+    private function getCleanedBranch(): string
+    {
+        $branch = $this->branch;
+        // e.g. pulls/4.10/some-bugfix or pulls/4/some-feature
+        // for push events to the creative-commoners account
+        if (preg_match('#^pulls/([0-9\.]+)/#', $branch, $matches)) {
+            $branch = $matches[1];
+        }
+        // e.g. 4.10-release
+        $branch = preg_replace('#^([0-9\.]+)-release$#', '$1', $branch);
+        return $branch;
+    }
     
-    private function parseBoolValue(mixed $value): bool
+    private function parseBoolValue($value): bool
```

Comment:
Why remove `mixed`?

Reply:
Not supported by PHP 7.4 which is what I'm running locally right now.

Reply:
Fair enough.